python/ray/serve/experimental/gradio_visualize_graph.py

```diff
@@ -232,6 +234,20 @@ def depths_fn(node):
         }
 
         with gr.Blocks() as demo:
+            from PIL import Image
+
+            try:
+                graph = _dag_to_dot(self.dag)
+                gr.Image(
+                    label="Direct Acyclic Graph",
+                    value=Image.open(BytesIO(graph.create(graph.prog, format="png"))),
+                )
+            except ModuleNotFoundError:
+                logger.warning(
+                    "Module `pydot` is not installed. Unable to show pydot "
```

Comment:
Do users need `graphviz` as well? Can we also include the `pip install` command needed to get the packages in this error message?

Reply:
Included both!